### ISSUE TITLE
Configurable/expandable column widths in task boards

### ISSUE THREAD

**Issue by haikalpribadi** [codetree/feedback#275]
The issue cards in the task boards can be a bit crowded sometimes. And there just seems to be so much space wasted on the side. See the screenshot below to get what I mean.

![CodeTree Task Board](https://user-images.githubusercontent.com/567679/45985788-d1dabe80-c05f-11e8-8df6-bed4f44645e0.png)

There's just so much space on the right side of the page, while the issue cards being cramped together.

Can we make the columns widths to be configurable/expandable?

**Comment by arifb** [comment]
This has been +1'd in our internal issue tracker, thanks for the feedback @haikalpribadi.

**Comment by haikalpribadi** [comment]
Do you have an idea of a timeline when it would be released? PS: I think there could be many who would like to contribute to the project if the codebase was open source ;)

**Comment by arifb** [comment]
@haikalpribadi No, we can't commit to a timeline for this feature, it's not at the top of the list. Also wouldn't hold your breath for us to open source the code, no plans for that. Sorry.
